modules/persistent-storage-csi-olm-driver-install.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc
+
+[id="persistent-storage-csi-olm-driver-install_{context}"]
+= Installing the {FeatureName} CSI driver
+
+The {FeatureName} CSI driver is not installed in {product-title} by default. Use the following procedure to install and configure this driver to enable the {FeatureName} CSI Driver Operator in your cluster.
+
+.Prerequisites
+* Access to the {product-title} web console.
+* Installed {FeatureName} CSI Operator.
+
+.Procedure
+To install the {FeatureName} CSI driver from the web console:
+
+. Log in to the web console.
+
+. Click *administration* -> *CustomResourceDefinitions* -> *ClusterCSIDriver*.
+
+. On the *Instances* tab, click *Create ClusterCSIDriver*.
+
+. Use the following YAML file:
++
+[source,file]
+----
+apiVersion: operator.openshift.io/v1
+kind: ClusterCSIDriver
+metadata:
+    name: efs.csi.aws.com
+spec:
+  managementState: Managed
+----
+
+. Click *Create*.
+
+. Wait for all of the Conditions to change to a "true" status.
```

Comment:
Wait for all *Available* Conditions to change to a "true" status. *Progressing* / *Degraded* should be "false".